opm/autodiff/BlackoilModelEbos.hpp

```diff
@@ -456,7 +456,10 @@ namespace Opm {
 
             for (int cell_idx = 0; cell_idx < nc; ++cell_idx) {
                 double dp = dx[cell_idx][flowPhaseToEbosCompIdx(0)];
-                reservoir_state.pressure()[cell_idx] -= dp;
+                //reservoir_state.pressure()[cell_idx] -= dp;
+                double& p = reservoir_state.pressure()[cell_idx];
+                p -= dp;
+                p = std::max(p, 1e5);
```

Comment:
My current experience tells me that limiting pressure updated is not essential. But I am working with @osae to understand and improve the Appleyard chopping. Expect more tuning related PRs in the future. I would not change this now.